nautobot/extras/forms/mixins.py

```diff
@@ -394,6 +400,33 @@ def clean(self):
           verify that the requested "source" object does not have an existing RelationshipAssociation to
           a different destination object.
         """
+
+        required_relationships = Relationship.objects.get_required_for_model(self.instance)
+        for side, relations in required_relationships.items():
+            for relation in relations:
+
+                if relation.symmetric:
+                    if not self.cleaned_data.get(f"cr_{relation.slug}__peer", []):
+                        model_name = relation.source_type.model_class()._meta.verbose_name
+                        # Only add the error once for this field
+                        if len(self.errors.get(f"cr_{relation.slug}__peer", [])) == 0:
+                            self.add_error(
+                                f"cr_{relation.slug}__peer", forms.ValidationError(f"You must select a {model_name}")
+                            )
+                    continue
```

Comment:
I didn't think we were going to implement/permit symmetric required relations?

Reply:
Ah yes https://github.com/nautobot/nautobot/issues/873#issuecomment-1217088337